components/ui/AppDataTable.vue

```diff
@@ -1,16 +1,26 @@
 <template>
-  <cv-data-table class="app-data-table" :columns="columns" :sortable="false">
-    <template #data>
+  <bx-table class="app-data-table" :columns="columns">
```

Comment:
Reading the [docs](https://www.ibm.com/standards/carbon/carbon-web-components/?path=/docs/components-data-table--default), it looks like we are missing the `<bx-data-table>` container between these two lines

Reply:
Added in https://github.com/Qiskit/qiskit.org/pull/3001/commits/12a325b0273ba5ecb35b9d752f830442a78b00a5.